playbooks/generate_discovery_iso_mce.yml

```diff
@@ -0,0 +1,5 @@
+---
+# file: generate_discovery_iso.yml
+- hosts: bastion

```

Comment:
This comment refers to 2 files. `playbooks/generate_discovery_iso_mce.yml` and [`generate_discovery_iso_mce/tasks/main.yml`](https://github.com/redhat-partner-solutions/crucible/pull/210/files#diff-88780b0ec67206c732cfa8128acb4f26e1658562eb0e78e8799e8ba20a6f86a3).

Discovery iso's are handled by the http_store host. The http_store is typically on the bastion host but not always. 
step 1: Please change the host to `http_store`.
step 2: in the `generate_discovery_iso_mce` role please delegate the `Download ISO` task to the `bastion`.

Reply:
done